django_feature_flip/tests.py

```diff
@@ -16,6 +17,14 @@ def test_max_of_40_characters_of_name(self):
         with self.assertRaises(DataError):
             FeatureFactory(name='This is a very large name, features should have short names')
 
+    def test_min_of_0_validation_for_time_percentage(self):
+        with self.assertRaises(ValidationError):
+            FeatureFactory(time_percentage=-1).clean_fields()
```

Comment:
I think here you are testing twice or the `clean_fields` is never called. Because  `FeatureFactory(time_percentage=-1)` should raise the exception since the factory creates it in the factory constructor. Maybe you can just build it first and the call `clean_fields`. wdyt?

Reply:
Actually these validations are not being rub when using the factory as they are meant to be run when using [forms (updating from the web) ](https://docs.djangoproject.com/en/3.0/ref/forms/validation/)

Reply:
Ok. Thanks for the explanation